### DISCUSSION TITLE
Day streak reset to 3 day but i didn't miss any day

### DISCUSSION THREAD

**Discussion by pullMyToe** [cssbattle/forum#390] [Bugs reports]
for some reason my day streak reset to 3 after 110 days + any way to fix this? thanks

**Comment by chinchang** [comment]
Hi there! There was a bug we are fixing right now. Can I have your username or profile link please?

**Reply by pullMyToe** [reply]
Great thanks
https://cssbattle.dev/player/tejoextremo

**Reply by chinchang** [reply]
it should be fixed now

**Comment by padi** [comment]
@chinchang any chance this is related to my streak breaking today? :/

My Dec 21 and 22 submission were definitely submitted on those days according to these screenshots:

<img width="1062" height="703" alt="Homepage Screenshot" src="https://github.com/user-attachments/assets/433e5170-7276-4850-8bf0-33d17f9fa038" />


<img width="1012" height="717" alt="Dec 21 2025 puzzle solutions" src="https://github.com/user-attachments/assets/e83454ac-22a8-41d0-b22f-bbcb727f6a90" />
<img width="991" height="725" alt="Dec 22 2025 puzzle solutions" src="https://github.com/user-attachments/assets/e5031df8-6f33-4f4e-946c-7bdfb071bcca" />

https://cssbattle.dev/player/mri if you need more info.
